cpp/ps0001/problem solve 1.cpp

```diff
@@ -1,19 +1,23 @@
 #include <iostream>
 using namespace std;
 int main(){
-    int first;
-    int second;
-    int third;
-    int sum;
-    cout <<"please enter first number\n";
-    cin >>first;
-    cout <<"please enter second number\n";
-    cin >>second;
-    cout <<"please enter third number\n";
-    cin >>third;
+    // prompting user for 3 numbers to add 1/2
+    short first;
+    short second;
+    short third;
+    short sum;
+    cout << "please enter first number\n";
+    cin >> first;
+    cout << "please enter second number\n";
+    cin >> second;
+    cout << "please enter third number\n";
+    cin >> third;
+
+    // outputting the result of the summation process 3/3
     sum = (first + second + third);
-    cout <<"sum : \n";
+    cout << "sum : \n";
     cout << sum;
 
-  return 0;
-}
\ No newline at end of file
+    return 0;
+}
+// great work 😎 8/10 ⭐

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors
- [x] no bugs occurs while execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code